pyopnsense/firewall.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Matthew Treinish
```

Comment:
Since you wrote this cod you can put your own copyright line here:

```suggestion
# Copyright 2022 Patrick Carr
```

All that was really missing before was the GPL header, the copyright lines themselves are less important IMO.